lib/safer_rails_console/patches/sandbox/auto_rollback.rb

```diff
@@ -25,8 +25,12 @@ def self.handle_and_reraise_exception(e)
         module ActiveRecord
           module ConnectionAdapters
             module PostgreSQLAdapter5
-              def execute_and_clear(sql, name, binds, prepare: false)
-                super
+              def execute_and_clear(sql, name, binds, prepare: false, async: false)
```

Comment:
Not your fault but can we generically capture the args to avoid any Rails version specific changes here i.e.

```ruby
def execute_and_clear(*, **)
  super
rescue => e
  # ...
end
```